homeassistant/components/unifi_direct/device_tracker.py

```diff
@@ -121,7 +121,7 @@ def _get_update(self):
 
 def _response_to_json(response):
     try:
-        json_response = json.loads(str(response)[31:-1].replace("\\", ""))
+        json_response = json.loads(str(response)[31:-1].replace("\\\\", "\\").replace("\\'", "'"))
```

Comment:
The device interface and data parsing code for this integration should be moved to a dedicated standalone library published on pypi.

https://developers.home-assistant.io/docs/creating_platform_code_review#5-communication-with-devicesservices

We won't except any changes to this integration until that is done.

Reply:
@MartinHjelmare the hill to die on here should not be moving this integration into a dedicated library and refusing changes until somebody does - it should be pressing the approve button on a 1 line change. The integration here has already been accepted & merged and the rest of us trying to use this integration are experiencing:
* inaccurate data
* thousands of errors
* crashing APs

As per https://github.com/home-assistant/core/issues/35284 (almost a year old)

This is code that obviously is no longer maintained so if you won't approve the fix then at least delete the integration and save us users from the misery and pain of trying to get this to work reliably. Others are already ripping this out as a custom component and applying this - stimulate them to move this into a git repo and we can use it via HACS.